wiki/People/The_Team/Account_support_team/id.md

```diff
@@ -25,17 +20,24 @@ Kamu harus menghubungi tim ini mengenai topik yang berhubungan dengan akun yang
 
 ### [accounts@ppy.sh](mailto:accounts@ppy.sh)
 
-- [Banding atas pembatasan yang diberlakukan pada akun kamu](/wiki/Help_Centre/Account_Restrictions), termasuk silence dan penghapusan konten profil.
+- [Banding atas pembatasan yang diberlakukan pada akun kamu](/wiki/Help_Centre/Account_Restrictions), termasuk:
+  - silence;
+  - penghapusan konten profil;
+  - penghapusan beatmap;
+  - penghapusan postingan dan komentar di forum.
 - Perilaku buruk yang [ingin kamu akui](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play#what-can-i-do-if-i've-broken-the-rules?).
 - [Kehilangan akses ke email](/wiki/Help_Centre#sign-in) yang terhubung dengan akun osu! kamu, atau saat akun kamu dicuri.
 - [Mengembalikan nama pengguna menjadi yang sebelumnya dan sedikit koreksi ejaan pada nama pengguna](/wiki/Help_Centre#name-changes).
 - [Pendaftaran akun bot](/wiki/Bot_Account).
 
 ### [support@ppy.sh](mailto:support@ppy.sh)
 
+*Catatan: Untuk melaporkan sebuah kasus seperti perilaku buruk dan pelecehan seksual yang ekstrim dari pengguna mana pun, termasuk dari seorang moderator, silakan kunjungi artikel [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour/Abuse).*
```

Comment:
`sebuah kasus` --> `kasus-kasus`

(itu contohnya ada lebih dari 1 soalnya)

Reply:
oke, diubah. kalo yang lainnya itu kaya yang di line 9 kah? `anggotanya` jadi `anggota-anggotanya` atau `masing-masing anggota`? udah cek di semua artikel, nyoba2 ctrl+F `sebuah` atau `suatu` atau `satu`, tapi nggak nemu yang lainnya lagi, dan udah cek ulang juga di masing2 artikel. takutnya ada yang miss D:

Reply:
imo semuanya applicable kok, tergantung konteks aja hehe

kalau yang dicontohkan itu cuma satu pakainya `sebuah (x)` / `suatu (x)` / `satu (x)`
kalau yang dicontohkan ada lebih dari satu pakainya `para anggotanya` / `seluruh anggotanya` / `anggota-anggotanya` / dsb

untuk line 9 itu sendiri coba pakai kalimat begini deh :

``Tim ini **hanya** dapat dihubungi melalui email pada alamat [support@ppy.sh](mailto:support@ppy.sh), [accounts@ppy.sh](mailto:accounts@ppy.sh), atau [privacy@ppy.sh](mailto:privacy@ppy.sh). Identitas para anggotanya sengaja dirahasiakan untuk melindungi mereka semua dari hal-hal yang tidak diinginkan.``

Reply:
also uhh `Point of Contact` nggak literally berarti `Titik Kontak` juga orz :(

mending dijadiin `Kontak` aja itu imo...

Reply:
hokkeee, all applied. udah recheck dan ada sedikit perubahan di paragraf awal2. thank you buat saran dan masukannya niva!